archunit/src/test/java/com/tngtech/archunit/core/importer/ImportOptionsTest.java

```diff
@@ -157,6 +161,30 @@ public void detects_archives_correctly(ImportOption doNotIncludeArchives) {
                 .isFalse();
     }
 
+    @DataProvider
+    public static Object[][] do_not_include_package_info_classes() {
+        return testForEach(new DoNotIncludePackageInfoClasses(), DO_NOT_INCLUDE_PACKAGE_INFO_CLASSES);
+    }
+
+    @Test
+    @UseDataProvider("do_not_include_package_info_classes")
+    public void detect_package_info_class(ImportOption doNotIncludePackageInfoClasses) {
+        NormalizedUri mockUri = mock(NormalizedUri.class);
+        StubLocation location = new StubLocation(mockUri);
+
+        given(mockUri.toString())
+                .willReturn("/path/to/package/and/file/package-info.class");
+        assertThat(doNotIncludePackageInfoClasses.includes(location))
+                .as("with path for package-info.class file")
```

Comment:
I think it's better if the description really refers to the argument of `assertThat(..)`. Because this way you would get an error like `with path for package-info.class file > expected false but was true`. But what should that mean? It would likely be better to have sth. like `doNotIncludePackageInfoClasses.includes(locationOfPackageInfo)`, then seeing `expected false but was true` makes sense, no? :thinking:

Reply:
yes, makes perfect sense.
fixed in https://github.com/TNG/ArchUnit/pull/794/commits/90c126f6acba68e0602e316c5d576001ee5f20ce